doc/users/next_whats_new/mpl_toolkit_pep420.rst

```diff
@@ -0,0 +1,9 @@
+``matplotlib.mpl_toolkits`` is now an implicit namespace package
+----------------------------------------------------------------
+
+Following the deprecation of ``pkg_resources.declare_namespace`` in ``setuptools`` 67.3.0,
+``matplotlib.mpl_toolkits`` is now implemented as an implicit namespace, following
+`PEP 420 <https://peps.python.org/pep-0420/>`_.
+
+This change should be backward compatible. In case of problems please `file an issue on GitHub
+<https://github.com/matplotlib/matplotlib/issues/new/choose>`_.
```

Comment:
While I sort of think it is a good idea to point directly to "New issue", it may still be better to point to the issues. If nothing else to avoid duplicates (if any...). But I'd wait and hear what other say.

Reply:
I think this is implicit (other wise would would put this on _every_ change note!).

Reply:
should I just drop line 8 and 9 then ?

Reply:
done !